docs/pyproject.toml

```diff
@@ -6,14 +6,92 @@ requires = ["poetry-core"]
 authors = ["Laurie Voss <github@seldo.com>"]
 description = ""
 name = "new-docs"
+package-mode = false
 readme = "README.md"
 version = "0.1.0"
 
 [tool.poetry.dependencies]
 python = "^3.11"
-mkdocs = "^1.5.3"
-mkdocstrings = {extras = ["python"], version = "^0.24.1"}
```

Comment:
guess we don't need this extra anymore?

Reply:
Good catch! It works because `mkcostrings-python` is still installed as a second-degree dependency somehow, but better be explicit.